lisp/mastodon-tl.el

```diff
@@ -140,6 +142,8 @@ Return value from boosted content if available."
                (format "(%s) " (propertize "F" 'face 'success)))
              (mastodon-tl--byline-author toot)
              (mastodon-tl--byline-boosted toot)
+             " posted: "
```

Comment:
I don't think `posted:` is necessary. Appending the timestamp is enough.

I think

```
| Orbfix (@orbfix@mstdn.io) 2017-04-27 15:59:04
 ------------
```

looks better (less cluttered) than

```
| Orbfix (@orbfix@mstdn.io) posted: 2017-04-27 15:59:04
 ------------
```